frontend/containers/settings-security/change-password/component.tsx

```diff
@@ -0,0 +1,127 @@
+import { FC } from 'react';
+
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Alert from 'components/alert';
+import Button from 'components/button';
+import ErrorMessage from 'components/forms/error-message';
+import Input from 'components/forms/input';
+import Label from 'components/forms/label';
+import Loading from 'components/loading';
+import { ChangePassword } from 'types/user';
+import { useChangePasswordResolver } from 'validations/change-password';
+
+import { useChangePassword } from 'services/users/userService';
+
+import { SettingsChangePasswordProps } from '.';
```

Comment:
It's probably better to avoid importing the types from the `index.ts` file since it also imports this component itself.

```suggestion
import { SettingsChangePasswordProps } from './types';
```